src/components/timeline/index.js

```diff
@@ -17,6 +17,11 @@ import {
   getZeroAt,
 } from '../../reducers/profile-view';
 import { getHiddenThreads, getThreadOrder } from '../../reducers/url-state';
+import { withSize } from '../shared/WithSize';
+import TimelineRuler from './Ruler';
```

Comment:
nit: can you put these two imports closer to the imports for the other components ? I try to keep the imports grouped by "type of thing".